app/models/manageiq/providers/google/cloud_manager/provision/cloning.rb

```diff
@@ -8,21 +8,23 @@ def do_clone_task_check(clone_task_ref)
     end
   end
 
-  def create_disk
-    source.with_provider_connection do |google|
-      return google.disks.create(
-        :name         => dest_name,
-        :size_gb      => get_option(:boot_disk_size).to_i,
-        :zone_name    => dest_availability_zone.ems_ref,
-        :source_image => source.name)
-    end
+  def initial_disk
+    {
+      :boot             => true,
+      :autoDelete       => true,
+      :initializeParams => {
+        :name        => dest_name,
+        :diskSizeGb  => get_option(:boot_disk_size).to_i,
+        :sourceImage => source.location,
```

Comment:
Yeah sorry, I forgot to add that to the commit.  Apparently the api for creating an instance (v1.compute.instance.insert) is more restrictive than the api for creating disks (v1.compute.disks.insert) - specifically, the 'sourceImage' parameter must be the full url of the image, not just the name.  We weren't persisting this information anywhere, so I added code to save it in the 'location' field on VmOrTemplate (which is similar to how the AWS provider operates).